requirements_all.txt

```diff
@@ -638,7 +638,7 @@ hbmqtt==0.9.5
 hdate==0.9.3
 
 # homeassistant.components.heatmiser
-heatmiserV3==0.9.1
+heatmiserV3==1.1.11
```

Comment:
Please run `script/gen_requirements_all.py` and commit the changes.

Reply:
Thanks for your help @MartinHjelmare  - that is now done.